library/postfixadmin

```diff
@@ -2,17 +2,17 @@
 Maintainers: David Goodwin <david@codepoets.co.uk> (@DavidGoodwin)
 GitRepo: https://github.com/postfixadmin/docker.git
 
-Tags: 3.2.2-apache, 3.2-apache, 3-apache, apache, 3.2.2, 3.2, 3, latest
+Tags: 3.2.3-apache, 3.2-apache, 3-apache, 3.2.3, 3.2, 3
```

Comment:
`apache` and `latest` tags removed on purpose? (and `fpm` and `fpm-alpine` removed below)

Reply:
https://github.com/postfixadmin/docker/blob/master/generate-stackbrew-library.sh#L60

Reply:
It seems that check in generate should be loosened; as far as I can tell `update-check.php` only concerns itself with the major and minor of the version and ignores patch level entirely:

http://postfixadmin.sourceforge.net/update-check.php?version=3.2.0 -> `latest`
http://postfixadmin.sourceforge.net/update-check.php?version=3.2.1 -> `latest`
http://postfixadmin.sourceforge.net/update-check.php?version=3.2.3 -> `latest`
http://postfixadmin.sourceforge.net/update-check.php?version=3.2.z -> `latest`
http://postfixadmin.sourceforge.net/update-check.php?version=3.3 -> `latest`
http://postfixadmin.sourceforge.net/update-check.php?version=4 -> `latest`

Reply:
:ping_pong: @DavidGoodwin